src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  display: flex;
+  margin: auto;
+  flex-wrap: wrap;
+  max-width: 800px;
+  min-width: 800px;
+
+  &__day {
+    display: flex;
+    height: $cell-size;
+    width: $cell-size;
+    background-color: $main-color;
+    border: 1px solid black;
+    font-family: $main-text-style;
+    font-size: $text-size;
+    transition-duration: 0.5s;
+    cursor: pointer;
+    justify-content: center;
+    align-items: center;
+    margin: 0.5px;
+
+    &:hover {
+      background-color: $hover-color;
+      transform: translateY(-20px);
+    }
+  }
+
+  @for $day from 1 through 31 {
+    .calendar__day--#{$day}::before {
+      content: "#{$day}";
+    }
+  }
+  
+  @each $name-day, $number-day in $days {
```

Comment:
when I am changing day in class nothing is changing

Reply:
fixed